package.json

```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "ceibo": "~2.0.0",
-    "ember-cli-babel": "^6.6.0",
+    "ember-cli-babel": "6.12.0",
```

Comment:
Can you prefix the version with a caret? `^6.12.0`. Or do we need to pin this specific version?

Reply:
👍